apps/jobs/urls.py

```diff
@@ -75,4 +75,9 @@
         views.get_bearer_token,
         name="get_bearer_token",
     ),
+    url(
+        r"^phase_split/(?P<challenge_phase_split_pk>[0-9]+)/team/(?P<participant_team_pk>[0-9]+)/github_badge/$",
```

Comment:
lets change the url to this 
`r"^phase_splits/(?P<challenge_phase_split_pk>[0-9]+)/teams/(?P<participant_team_pk>[0-9]+)/github_badge/$"`

Reply:
Notice the `s` at the end for phase split and team.